example/.travis.yml

```diff
@@ -102,13 +102,21 @@ env:
     # - ORCA_PHPCS_STANDARD=AcquiaDrupalTransitional
     #
     # Enable Nightwatch.js testing. Acceptable values are "TRUE" and "FALSE".
-    # - ORCA_ENABLE_NIGHTWATCH=TRUE
+    - ORCA_ENABLE_NIGHTWATCH=TRUE
     #
     # To enable telemetry with Amplitude on cron runs, uncomment the following
     # line and set ORCA_AMPLITUDE_API_KEY in your Travis CI repository settings:
     # @see https://github.com/acquia/orca/blob/main/docs/advanced-usage.md#ORCA_TELEMETRY_ENABLE
     # @see https://github.com/acquia/orca/blob/main/docs/advanced-usage.md#ORCA_AMPLITUDE_API_KEY
     # - ORCA_TELEMETRY_ENABLE=TRUE
+    #
+    # Change the path where ORCA looks for the SUT. Accepted values are any valid directory path,
+    # e.g., /var/www/example, or ../example.
+    - ORCA_SUT_DIR=${TRAVIS_BUILD_DIR}/example
+    #
+    # Change the path where ORCA stores Clover file,
+    # e.g., logs/clover-self.xml
+    - ORCA_COVERAGE_CLOVER=$HOME/build/logs/clover-self.xml
```

Comment:
This seems out of scope for this PR. Did you mean to add it?

Reply:
I will keep it commented